routes/messages.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router = express.Router();
+const messages = require('../Messages')
+
+
+
+
+// - [ ] Read all messages
+
+router.get("/", (request, response) => {
+    response.json(messages)
+});
+
+// - [ ] Read all messages
+
+router.get("/latest", (request, response) => {
+    response.json(messages.slice(-10))
```

Comment:
whil you still get latest message after you post new one? the messages variable is outside?